RecoTracker/TkTrackingRegions/plugins/PointSeededTrackingRegionsProducer.h

```diff
@@ -202,10 +202,10 @@ class  PointSeededTrackingRegionsProducer : public TrackingRegionProducer
     int n_regions = 0;
     for(size_t i = 0; i < n_points && n_regions < m_maxNRegions; ++i ) {
 
-      double x = TMath::Cos(phiPoints[i]);
-      double y = TMath::Sin(phiPoints[i]);
-      double theta = 2*TMath::ATan(TMath::Exp(-etaPoints[i]));
-      double z = (x*x+y*y)/TMath::Tan(theta);
+      double x = std::cos(phiPoints[i]);
+      double y = std::sin(phiPoints[i]);
+      double theta = 2*std::atan(std::exp(-etaPoints[i]));
+      double z = 1./std::tan(theta);
```

Comment:
how about
z = ( 1−exp(-η)² ) / (2 * exp(-η))
?

No idea whether it is more accurate and/or faster or neither...